README.rst

```diff
@@ -127,7 +127,8 @@ The following commands are supported:
 * ``status`` — get current instance(s) status;
 * ``log`` — get logs of instance(s);
 * ``clean`` - clean instance(s) files;
-* ``pack`` — pack the application into a distributable bundle.
+* ``pack`` — pack the application into a distributable bundle;
+* ``repair`` — repair the cluster.
```

Comment:
I believe it should be extended. It's not clear that means "repair the cluster"

Reply:
What about `Patch cluster-wide config file`?

Reply:
"Patch cluster configuration files"